src/main/java/homework_nr_8/Main.java

```diff
@@ -0,0 +1,36 @@
+package homework_nr_8;
+
+public class Main {
+    public static void main(String[] args) {
+        Appliance app_1 = new WashingMachine("LG", 2000, 7.5);
+        Appliance app_2 = new Microwave("Samsung", 1200, true);
+        Appliance app_3 = new AirConditioner("Daikin", 1500, 12000, true);
+
+        Appliance[] appliances = new Appliance[]{
+                app_1, app_2, app_3
+        };
+        for (int i=0; i !=appliances.length; i++){
+            System.out.println(appliances[i].displayInfo());
+            if (appliances[i] instanceof WashingMachine) {
```

Comment:
Use the better syntax of instanceof `element instanceOf Type castedBufferElement`